lib/travis/build/script/node_js.rb

```diff
@@ -263,6 +226,18 @@ def prepend_path(path)
           def yarn_req_not_met
             "$(travis_vers2int $(echo `node --version` | tr -d 'v')) -lt $(travis_vers2int #{YARN_REQUIRED_NODE_VERSION})"
           end
+
+          def is_win?
+            config[:os].downcase.strip == 'windows'
```

Comment:
Can we rely on `config[:os]` to be sure of the operating system where the script will run?

Reply:
I'm open to suggestions.

Reply:
@BanzaiMan Why does a runtime `${TRAVIS_OS_NAME}` check mean that nvm and nvs need to be present on all images? :confused:

Reply:
@meatballhat It appears my thinking was wrong.

Reply:
I think we should decide which version manager to use at the script compile time. Otherwise we need to check `${TRAVIS_OS_NAME}` value everywhere.